CourtBallGame/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+from court.models import Court
+
+MAX_LEN_BALL_GAME_STR = 100
+
+
+class BallGame(models.TextChoices):
+    Soccer = 'Soccer', 'Soccer'
+    Basketball = 'Basketball', 'Basketball'
+    Volleyball = 'Volleyball', 'Volleyball'
+    Baseball = 'Baseball', 'Baseball'
+    Tennis = 'Tennis', 'Tennis'
+    Rugby = 'Rugby', 'Rugby'
+    Golf = 'Golf', 'Golf'
+    Cricket = 'Cricket', 'Cricket'
+    Handball = 'Handball', 'Handball'
+
+
+class CourtBallGame(models.Model):
+    court_id = models.ForeignKey(Court, on_delete=models.CASCADE)
+    ball_game = models.CharField(
+        max_length=MAX_LEN_BALL_GAME_STR,
+        choices=BallGame.choices,
+        blank=False
```

Comment:
Default value of blank is set to false, so you can remove it if you want to.

Reply:
blank=False is a parameter that can be passed to CharField to indicate that the field is required and cannot be left blank.
I think it is required here.

Reply:
Got your tip, Its a default parameter.